### PR TITLE
Add additional json.dumps() arguments via JSON_OPTIONS setting

### PR DESCRIPTION
**Pull request by aptwebapps**
I implemented the JSON_OPTIONS settings we discussed back in February. If you like what you see I can add a Readme update.

### CODE REVIEW

**Comment by aptwebapps** [comment]
I forgot to handle the exception I raised (if using a dot-string cls value) for versions of Django earlier than 1.7. So of course Travis didn't like it. Now fixing Python 3 issues.

**Comment by aptwebapps** [comment]
Also, I'm not happy with the NotImplementedError there but couldn't think of a more appropriate one, except maybe ImportError itself.

**Comment by jsocol** [comment]
I've got a couple of notes—I'm going to just take care of them myself, but I'm writing them down here so I don't forget

File: jsonview/decorators.py [hunk 1]
```diff
@@ -30,9 +30,23 @@
 
 
 def _dump_json(data):
+    options = getattr(settings, 'JSON_OPTIONS', {})
+
     if getattr(settings, 'JSON_USE_DJANGO_SERIALIZER', True):
```
Comment (jsocol) [inline jsonview/decorators.py]: should prioritize `options` over `JSON_USE_DJANGO_SERIALIZER` and add a `DeprecationWarning` to using this arg—`JSON_OPTIONS` is strictly more powerful and is The Future™

File: jsonview/decorators.py [hunk 2]
```diff
@@ -30,9 +30,23 @@
...
-        return json.dumps(data, cls=DjangoJSONEncoder)
-    return json.dumps(data)
+        options['cls'] = DjangoJSONEncoder
+    else:
+        if 'cls' in options and type(options['cls']) in (str, unicode):
```
Comment (jsocol) [inline jsonview/decorators.py]: gonna use `isinstance(options['cls'], six.string_types)`

File: jsonview/decorators.py [hunk 3]
```diff
@@ -30,9 +30,23 @@
...
+            try:
+                from django.utils.module_loading import import_string
+            except ImportError:
+                raise NotImplementedError(
```
Comment (jsocol) [inline jsonview/decorators.py]: can fallback to importlib

**Comment by jsocol** [comment]
Rebased and merged in along with 60fefdd264d91d6f87fb35aad09b08f3a596563d

**Comment by jsocol** [comment]
Thanks for getting it started, @aptwebapps!

**Pull request closed without merging**